eng/pipelines/templates/jobs/ci.mgmt.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           targetPath: $(Build.ArtifactStagingDirectory)
       - pwsh: |
           $releaseTag = "FOR_SIGNING"
```

Comment:
These tags are not really needed after this change as those tags are only used by release pipelines.

Reply:
I will look to see if it's no longer required, and clean it up.